src/util/Spotify.js

```diff
@@ -48,53 +48,66 @@ export const Spotify = {
         Tracklist.js, cannot read property 'map' of undefined, and pointing on the
         traks of the TrackList.. I couldn't understand why.*/
     async search (term) {
-      /* let userAccessToken = this.getUserAccessToken();   I commented this out becaues it is already defined in the global of this file*/
+      
+      const topic = 'Searching Spotify\'s API.'
+
+      const mainTerm = encodeURI(term); // this converts the term passed as a parameter to a URI format

```

Comment:
naming; a better name would be `encodedTerm` 

since `main` doesnt mean anything in this context

also the naming `term` is unfamiliar, to me at least, do you mean `userSearchQuery` ?

Reply:
Term as in the term which is being searched a search term. I changed it from mainTerm to searchTermURI

Reply:
The rational was a term to search for as in "search term" but probabaly in this context it is too generalized so I changed it to searchTermURI.